opus/application/static_media/js/opus.js

```diff
@@ -55,11 +55,12 @@ var opus = {
     colLabelsNoUnits: [], // store labels without units (similar to data in colLabels but no units )
 
     // searching - making queries
-    selections:{},        // the user's search
-    extras:{},            // extras to the query, carries units, string_selects, qtypes, size, refreshes result count!!
+    selections: {},        // the user's search
+    extras: {},            // extras to the query, carries units, string_selects, qtypes, size, refreshes result count!!
     lastSelections: {},   // last_ are used to monitor changes
     lastExtras: {},
-    resultCount:0,
+    resultCount: 0,
+    maxBufferSize: 1000,    // max number of obserations to store in cache; at some point, we can probably figure this out dynamically
```

Comment:
I think "buffer" is too generic a name for this. The can be lots of buffers. How about maxCachedObservations or something like that?

Reply:
ah good suggestion; i didn't like the name but couldn't come up w/something better in the moment.